### PR TITLE
feat:(test)

### PR DESCRIPTION
**Pull request by Emigrate**

### CODE REVIEW

File: task4/index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>task-4</title>
+</head>
+<body id="bg" class="background">
+<div class="column">
+
+    <!-- LEVIY RYADOK -->
```
Comment (Sacronux) [inline task4/index.html:16]: comments must be on english

File: task4/index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    <div class="left-column">
+        <div class="box-main-1">
+            <div class="box-1">
```
Comment (Sacronux) [inline task4/index.html:20]: get rid of such class names, it is not reusable. Use more declarative names, e.g. 'purple-box', 'border-container'

File: task4/index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            </h2>
+            <div class="divider divider-3"></div>
+            <div class="btn-container-3">
+                <a class="btn-3">No</a>
+                <a class="btn-3">Yes</a>
+            </div>
+        </div>
+    </div>
+
+    <!-- PRAVIY RYADOK -->
```
Comment (Sacronux) [inline task4/index.html:57]: comments must be on english

File: task4/style.css [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+    width: 344px;
+    height: 224px;
+    margin-top: 82px;
+    margin-left: 27px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 2px;
+}
+
+.box-2 {
```
Comment (Sacronux) [inline task4/style.css:78]: duplication of styles `.box-1`, create class `.box` and classes for background colors

File: task4/style.css [hunk 5]
```diff
@@ -0,0 +1,510 @@
...
+.btn-container-1 {
+    display: flex;
+    margin-top: 8px;
+    margin-left: 1px;
+    padding-left: 11px;
+}
+
+.btn-1 {
+    text-transform:uppercase;
+    margin-top: 9.5px;
```
Comment (Sacronux) [inline task4/style.css:105]: how can you imagine half of the pixel on your screen?

File: task4/style.css [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    color: #009688;
+}
+
+.divider {
+    margin-top: 22px;
+    width: 344px;
+    height: 0.1px;
```
Comment (Sacronux) [inline task4/style.css:136]: pixels are not divisible measurements

File: task4/style.css [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+    margin-left: 8px;
+}
+
+.box-4 {
+    width: 168px;
+    height: 168px;
+    background: #009688;
+}
+
+.h2-box4 {
```
Comment (Sacronux) [inline task4/style.css:207]: duplication of styles `.h2-box3`

File: task4/style.css [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+}
+
+.box-3 {
+    width: 168px;
+    height: 168px;
+    background: #3E50B4;
+}
+
+.h2-box3 {
+    margin-top: -61px;
```
Comment (Sacronux) [inline task4/style.css:182]: if we need to use negative margins in such simple layout - we probably do something wrong

**Review by crucas — CHANGES_REQUESTED** [review]

**Comment by crucas** [comment]
Too many duplications of styles, get rid of digits in class names, don't divide pixels, don't use negative numbers in margins (I really don't think we need it's there)

File: task4/style.css [hunk 9]
```diff
@@ -0,0 +1,371 @@
...
+    margin-top: 64px;
+}
+
+.l-column {
+    display: flex;
+    margin-top: 25px;
+    flex-direction: column;
+}
+
+.r-column{
```
Comment (Sacronux) [inline task4/style.css:107]: duplication of styles `.l-column`

**Review by crucas — CHANGES_REQUESTED** [review]

**Review by crucas — APPROVED** [review]

**Pull request merged**